IPython/core/tests/test_async_helpers.py

```diff
@@ -9,6 +9,12 @@
 from unittest import TestCase
 from IPython.testing.decorators import skip_without
 import sys
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from IPython import get_ipython
+    ip = get_ipython()
```

Comment:
```suggestion

    ip = get_ipython()
```

To make corde formatter happy.